cargo-dist/src/cli.rs

```diff
@@ -54,23 +54,23 @@ pub struct Cli {
     ///
     /// If left unspecified we will use the values in [workspace.metadata.dist],
     /// except for `cargo dist init` which will select some "good defaults" for you.
-    #[clap(long, short)]
+    #[clap(long, short, value_delimiter(','))]
     #[clap(help_heading = "GLOBAL OPTIONS", global = true)]
     pub target: Vec<String>,
 
     /// Installers we want to build
     ///
     /// If left unspecified we will use the values in [workspace.metadata.dist].
     ///  `cargo dist init` will persist the values you pass to that location.
-    #[clap(long, short)]
+    #[clap(long, short, value_delimiter(','))]
     #[clap(help_heading = "GLOBAL OPTIONS", global = true)]
     pub installer: Vec<InstallerStyle>,
 
     /// CI we want to support
     ///
     /// If left unspecified we will use the value in [workspace.metadata.dist].
     /// `cargo dist init` will persist the values you pass to that location.
-    #[clap(long, short)]
+    #[clap(long, short, value_delimiter(','))]
     #[clap(help_heading = "GLOBAL OPTIONS", global = true)]
     pub ci: Vec<CiStyle>,
```

Comment:
unfortunately not entirely sure if we can get away with these, given `generate` supports global args [and has positional args](https://opensource.axo.dev/cargo-dist/book/reference/cli.html#cargo-dist-generate)... but maybe `generate` shouldn't take global args, now that I think about it? cc @mistydemeo 

context for misty: value_delimiter unlocks two new arg passing modes, in addition the one we get by default with Vec:

```
--my-vals=a --my-vals=b
--my-vals=a,b
--my-vals a b
```

The last one is problematic if your Command takes any positional args (is "b" part of `--my-vals` or a positional arg?). This is why that arg-passing mode is disabled by default in modern clap.

We largely don't use positional args anymore, except in `generate`, but generate doesn't really have a reason to take the global args anyway (filtering it down to a subset of platforms would almost surely be an error). (Alternatively it's possible it's "fine" anyway because maybe clap can disambiguate based on a GenerateStyle never parsing as any of these..?)

Reply:
`generate`'s args are largely not used by other people, just us, right? I think we can adjust its behaviour without impacting our users.